database/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM frankinaustin/postgis-multiarch:14-3.3
+dFROM frankinaustin/postgis-multiarch:14-3.3
 RUN apt-get update
-RUN apt-get install -y aptitude magic-wormhole vim
```

Comment:
i am not a wormhole user but i realize others might find it important to keep this in the container. i was not up for troubleshooting this build issue. let me know what y'all prefer I do 🙏